subprojects/publish/src/main/java/org/gradle/api/publish/internal/component/package-info.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 the original author or authors.
+ * Copyright 2023 the original author or authors.
```

Comment:
Yeah This one is weird. Not sure why git decided to think this was a rename. Probably because the file header is the same, and the content of the class is so small. I'm not sure how to resolve this, but I also don't think its really that big of a deal.

Reply:
Yeah, I don't think it is a problem, I just wanted to point it out.